### PR TITLE
fix: populate 'winners' array during payments

### PR DESCRIPTION
**Pull request by narekcat**

### CODE REVIEW

**Review by maxceem — CHANGES_REQUESTED** [review]
@narekcat we would like to use another endpoint for getting userId by handle `GET /v3/members/{handle}` and create for this another method `getV3MemberDetailsByHandle`.

We don't want to use method `getMemberDetailsByHandles` because for some users it fails, see the issue details:

image

**Comment by narekcat** [comment]
@maxceem . Ok, sorry it's my bad. I'll fix it.

**Comment by narekcat** [comment]
@maxceem I have fixed this issue, please check again.

File: src/common/helper.js [hunk 1]
```diff
@@ -1097,6 +1097,22 @@ async function getMemberDetailsByHandles (handles) {
   return _.get(res.body, 'result.content')
 }
 
+/**
+ * Get topcoder member details by handle.
+ *
+ * @param {Array} handle the user handle
+ * @returns {Array} the member details
```
Comment (maxceem) [inline src/common/helper.js:1104]: Params type `Array` doesn't match the actual data type.

**Review by maxceem — CHANGES_REQUESTED** [review]
Thank you @narekcat, work good now.

There is just one minor thing in the comment below.

File: src/common/helper.js (hunk 1 above)
Comment (narekcat) [reply src/common/helper.js]: I have fixed this issue too.

**Pull request merged**